### ISSUE TITLE
bpl_libraries

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-facilities-tmp#35]
- [x] s3
- [ ] py
- [ ] sql

https://data.cityofnewyork.us/Recreation/BPL-Branches/xmzf-uf2w

**Comment by SPTKL** [comment]
https://db-data-recipes.sfo2.digitaloceanspaces.com/pipelines/db-facilities/2019-05-14/datapackage.json

**Comment by baolingz** [comment]
For the BPL dataset, the dataset in s3 is a json file which I am not sure how to load into the workflow. I found an identical csv version for the dataset, could we use this one instead?

For the BPL dataset, the dataset in s3 is a json file which I am not sure how to load into the workflow. I found an identical csv version for the dataset, could we use this one instead?

https://data.cityofnewyork.us/Recreation/BPL-Branches/xmzf-uf2w

There's a csv attachment at the bottom of the page
https://data.cityofnewyork.us/api/views/xmzf-uf2w/files/20a12ae9-bf36-4125-a38d-2287e8e9ec5d?download=true&filename=BPL_Locations.csv
